src/component/Book.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Delete from './Delete';
+import '../CSS/book.css';
+
+const Book = () => (
+  <div className="book-container">
+    <div>
+      <p>Action</p>
+      <h3>Hunger Games</h3>
```

Comment:
Great job setting up React components for the Bookstore. You are off a great start in creating impressive React applications 💯 

As per project requirements, to improve the re-usability of the book component it must only render one book. Kindly consider the following suggestions:

- The `<Book />` component takes the book data as a prop and renders the title and the author, as well as the buttons without functionality for now. You can use a books array to map many books to the `<Book />` component.

i.e

```jsx
const Book = ({ category, author, title }) => (
  <div className="book-container">
    <div>
      <p>{category}</p>
      <h3>{title}</h3>
      <small>{author}</small>
      <div>
        <button type="button">Comments</button>
        <Delete />
        <button type="button">Edit</button>
      </div>
    </div>
    {/* ... rest of the code */}
  </div>
);
```

- Create a `<BookList />` component that renders a list of books using the `<Book>` component . This way adheres to one of the (S)OLID principles - (S)ingle functionality, since the the Booklist has only one functionality which is to render a list of books.

```jsx
const BookList = () => {
  // Array of book data
  const books = [
    {
      author: 'Author 1',
      title: 'Book 1',
      category: 'Category A',
    },
    {
      author: 'Author 2',
      title: 'Book 2',
      category: 'Category B',
    },
    {
      author: 'Author 3',
      title: 'Book 3',
      category: 'Category C',
    },
  ];

  return (
    <div className="book-list">
      {books.map((book, index) => (
        <Book
          key={index}
          author={book.author}
          title={book.title}
          category={book.category}
        />
      ))}
    </div>
  );
}

export default BookList;
```

- Then finally, render all your components on the <Books /> component ( `<BookList />` and `<CreateForm />` ). This also supports the single functionality rule because it acts as a page that only renders different components to create different content sections.